docs/aws-partner-addons.md

```diff
@@ -4,6 +4,7 @@ The following addons are provided by [AWS Partners](https://aws.amazon.com/partn
 
 | Addon | Description |
 |-------|:------------|
+| [Amazon FSx for NetApp ONTAP CSI driver](https://github.com/NetApp/terraform-aws-netapp-fsxn-eks-addon) | NetApp's Astra Trident provides dynamic storage orchestration for FSx for NetApp ONTAP using a Container Storage Interface (CSI) compliant driver. |
```

Comment:
And please add to the bottom of the list

```suggestion
| [NetApp - Trident Operator](https://github.com/NetApp/terraform-aws-netapp-fsxn-eks-addon) | NetApp's Astra Trident provides dynamic storage orchestration for FSx for NetApp ONTAP using a Container Storage Interface (CSI) compliant driver. |
```

Reply:
@bryantbiggs I've changed the name a bit from your suggestion and pushed it down the the bottom row. added in another commit

Reply:
looks good, thank you!